.github/workflows/no-exception-workflow.yml

```diff
@@ -5,11 +5,68 @@ on:
     branches: [ master ]
 
 jobs:
-  run:
-    name: Build
+  no-exception-lucene-and-others-javadoc:
     runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Set up JDK 8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 8
+
+    - name: Install checkstyle
+      run: mvn -e --no-transfer-progress clean install -Pno-validations
+
+    - name: Install dependencies
+      run: sudo apt install groovy
+
+    - run: ./.ci/no-exception-test.sh no-exception-lucene-and-others-javadoc
+
+  no-exception-cassandra-storm-tapestry-javadoc:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Set up JDK 8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 8
+
+    - name: Install checkstyle
+      run: mvn -e --no-transfer-progress clean install -Pno-validations
+
+    - name: Install dependencies
+      run: sudo apt install groovy
+
+    - run: ./.ci/no-exception-test.sh no-exception-cassandra-storm-tapestry-javadoc
```

Comment:
please make order of runs like:
`Set up JDK 8`
`Install dependencies`
`Checkout repository`
`Install checkstyle`
` ./.ci/no-exception-test.sh`

Reply:
Done